core/src/main/java/org/commonjava/maven/ext/core/impl/DependencyManipulator.java

```diff
@@ -430,6 +432,11 @@ private void apply( final Project project, final Model model, final Map<Artifact
                     dependencyManagement.getDependencies().addAll( 0, extraDeps );
                 }
             }
+            else if ( commonState.isOverrideTransitive() && dependencyState.getRemoteBOMDepMgmt() == null )
+            {
+                logger.warn( "Ignoring {}={} since it was used without the {} option", "overrideTransitive",
+                        commonState.isOverrideTransitive(), "dependencyManagement" );
```

Comment:
Why are we parametrising a hard-coded string?  What about:
```
 logger.warn( "Ignoring overrideTransitive=true since it was used without the dependencyManagement option");
```

Reply:
I was hoping that there were constants somewhere that gave the properties for the options and then I would fill them in from that. But I guess not? Or maybe they're not `public`?

Reply:
Please look at dc6017d to see what I would like to do.

The `true` part could be hardcoded or even left out, but I think that these property names should all be `public static final` constants so that they can be referenced elsewhere in the code. I checked and some are `public`, but most are `private` to that class.

It's probably best done as a separate commit. I changed only the two classes I needed for this commit.

Reply:
Pros/cons - you probably negate any performance change in your work elsewhere via the extra string 'appends'. However, there is an argument to use the reference to the constant thereby eliminating any issues with future renaming. I guess any impact is minimal though I would still like to see the boolean parameter got rid of.

Reply:
Should it even have the "=true" in the message or just "Ignoring {} option"? I used the word option but I wasn't sure if that's what we're calling them (or "property" might be better).

Reply:
Yeah you could remove the =true I guess.
How about 'flag' ? I think option is actually a bit better than property ; its a setting on PME after all.

Reply:
Done in f632569.